CONTRIBUTING.md

```diff
@@ -29,7 +29,12 @@ through our [GitHub issues page](http://github.com/ni/nidaqmx-python/issues).
    > The codegen scripts require Python 3.8 or later.
 8. Run all the regression tests again (including the tests you just added), and confirm that they all
 pass.
-9. Send a GitHub Pull Request to the main repository's master branch. GitHub Pull Requests are the
+9. Run `poetry run ni-python-styleguide lint` to check that the updated code follows NI's Python coding
+conventions. If this reports errors, first run `poetry run ni-python-styleguide fix` in order to sort
+imports and format the code with Black, then manually fix any remaining errors.
+10. Run `poetry run mypy generated/nidaqmx` and `poetry run mypy tests` to statically type-check the
```

Comment:
Should we set `pyproject.tom:tool.mypy:files` to `generated/nidaqmx/*.py,generated/nidaqmx/**/*.py,tests/*.py,tests/**/*.py`
so that these two command can combine to `poetry run mypy`?

Reply:
Yes, that would make sense.

Reply:
FYI: 
- `a/*.py,a/**/*.py` is redundant because `a/**/*.py` matches `a/b.py`
-  Globbing for `*.py` excludes `.pyi` stubs. Specifying directories does not.
- This catches some test problems that mypy ignored when we checked the module and tests separately :(

Reply:
Interesting..., our default config (what our internal template gives you) is `**/*.py` to hit both the module and tests (I went and read the [docs](https://mypy.readthedocs.io/en/stable/config_file.html#confval-files) this morning and they seemed to be indicating that `**/` would not match 0 length. If it works then great.

Reply:
I tried calling glob("tests/**/*.py") directly and now I'm not sure.